Lib/test/_test_multiprocessing.py

```diff
@@ -2705,6 +2705,7 @@ def test_terminate(self):
         p = self.Pool(3)
         args = [sleep_time for i in range(10_000)]
         result = p.map_async(time.sleep, args, chunksize=1)
+        time.sleep(0.2)  # give some tasks a chance to start
```

Comment:
Less is better, because it affects the total testing time (multiplied by 12 for all combinations).

0.1 was not enough to start all tasks in test_managers on my computer.

Reply:
This makes sense to me. It's a complicated situation, this makes it more likely that the test actually winds up testing what it is designed to. Sometimes that is the most we can ask for. It is much more likely to find a regression this way.